test/functional/forknotify.py

```diff
@@ -41,11 +42,17 @@ def run_test(self):
         self.nodes[1].generate(1)
         self.sync_all()
 
-        with open(self.alert_filename, 'r', encoding='utf8') as f:
-            alert_text = f.read()
+        # Give bitcoind 10 seconds to write the alert notification
+        timeout = 10
```

Comment:
This should rarely actually take 10 seconds, if the code is correct, right? Maybe 1 second will be a bit better in the case where it is actually broken & you're trying to fix it.